lib/logstash/inputs/gelf.rb

```diff
@@ -68,20 +79,32 @@ def initialize(params)
   public
   def register
     require 'gelfd'
+    @tcp = nil
+    @udp = nil
   end # def register
```

Comment:
Why do we need to set these to nil?

Wont using use_udp and use_tcp disable that protocol without having to do this?